### PR TITLE
Allow access to objects in the state bucket

### PR DESCRIPTION
**Pull request by adys**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -68,15 +68,15 @@ data "aws_iam_policy_document" "state_policy_root" {
...
+      for policy in concat(var.state_policies, local.bucket_policies) : policy.statement
     ])
 
     content {
       sid         = lookup(statement.value, "sid", null)
       effect      = lookup(statement.value, "effect", null)
       actions     = lookup(statement.value, "actions", null)
       not_actions = lookup(statement.value, "not_actions", null)
-      resources   = [local.bucket_arn]
+      resources   = [local.bucket_arn, "${local.bucket_arn}/*"]
```
Comment (dojci) [inline main.tf]: update code style to be uniform, use multiline and format (like elsewhere in the code)

Comment (adys) [reply main.tf]: could you check whether it's aligned properly now?

File: main.tf [hunk 2]
```diff
@@ -76,7 +76,11 @@ data "aws_iam_policy_document" "state_bucket_policy" {
       effect      = lookup(statement.value, "effect", null)
       actions     = lookup(statement.value, "actions", null)
       not_actions = lookup(statement.value, "not_actions", null)
-      resources   = [local.bucket_arn, "${local.bucket_arn}/*"]
+
+      resources = [
+        local.bucket_arn,
+        "${local.bucket_arn}/*"
```
Comment (dojci) [inline main.tf]: it would still be nice to use the format() function rather than using string interpolation syntax

Comment (adys) [reply main.tf]: Sorry I though you meant t fmt. Now I get it and it makes sense. Could you check again?

**Review by dojci — APPROVED** [review]
lgtm

**Pull request merged**